projects/canopy/src/lib/header/header.component.scss

```diff
@@ -2,15 +2,27 @@
 
 .lg-header {
   background-color: var(--header-bg-color);
-  box-shadow: var(--bottom-shadow);
+  box-shadow: var(--header-shadow);
+  left: 0;
+  position: fixed;
+  right: 0;
+  top: 0;
+  height: var(--header-height);
```

Comment:
In the FEP we have stylelint order our properties for us alphabetically, but I guess we don't in Canopy. Could we check the new properties are sorted correctly?

Reply:
(not a deal breaker for me, since we can always add the stylelint rule in later)

Reply:
I ahve sorted the files I have touched... will raise a ticket to add that rule to stylelint